src/assets/sass/canya.scss

```diff
@@ -43,3 +43,203 @@
 // @import "sass/theme/uikit/nav";
 // @import "sass/theme/uikit/alert";
 // @import "sass/theme/layout/topbar";
+
+h1, h2, h3{
+    font-weight: 700;
+}
+h4{
+    font-weight: 600;
+}
+
+.text-center{
+    text-align: center;
+}
+
+.text-left{
+    text-align: left;
+}
+
+.text-right{
+    text-align: right;
+}
+
+
+/* Padding classes */
```

Comment:
These classes seem helpful at first, but they fill the final CSS with many unused classes. 

Bootstrap 4 already include these in its utilities/_spacing.scss http://getbootstrap.com/docs/4.1/utilities/spacing/

Also, we have a function available: `get-spacer(1..5)` that you can use: 

```
padding: get-spacer(3) get-spacer(4);
```

To be more specific with your HTML elements spacing. This will avoid having lots of `.mt-*` classes  in the elements.

Reply:
@wie-canya